test/DotNetEnv.Tests/EnvTests.cs

```diff
@@ -2,6 +2,7 @@
 using System.Linq;
 using System.IO;
 using System.Runtime.InteropServices;
+using DotNetEnv.Extensions;
```

Comment:
Add a new set of tests somewhere for all 3 cases.

Reply:
I will add them soon. Just waited for clarification about how to go on ;)
I will also elaborate the new feature in the Readme in the next days.

Reply:
Tests added

Reply:
It definitely makes me sad that people need to add an extra using to include these. I would prefer to avoid that. Or do you think this approach (an explicitly separate `using Extensions`) is more the normal convention?

Reply:
I personally do not really mind about usings... They are just "hidden" statements at the top of my files, and my IDE handles them, I just have to do something if there are potential duplicates for example. And discovering available methods from unused namespaces is something the IDE handles quite good too.

So for me, it is more adorable to have Classes matching folders/files, and having a "structurated" Folder-approach, which helps understanding the structure of the project.

But if you prefer it to be different I will remove the folder and place the files directly within the root.